python/ray/worker.py

```diff
@@ -73,6 +72,15 @@
 # using logging.basicConfig in its entry/init points.
 logger = logging.getLogger(__name__)
 
+try:
+    import setproctitle
+except ImportError:
+    setproctitle = None
+    logger.warning(
```

Comment:
This might be a bit verbose if you have a lot of workers, but it is by intention.